charts/consul/test/acceptance/framework/consul/consul_cluster.go

```diff
@@ -356,7 +356,7 @@ func configurePodSecurityPolicies(t *testing.T, client kubernetes.Interface, cfg
 					Name: "test-psp",
 				},
 				Spec: policyv1beta.PodSecurityPolicySpec{
-					Privileged:          false,
+					Privileged:          true,
```

Comment:
driveby curious: was this the fix to allow psp's to work again? Or was there another PR as well?